### DISCUSSION TITLE
Lesson 5: brownie command-line results in nothing

### DISCUSSION THREAD

**Discussion by Guanxinyuan** [smartcontractkit/full-blockchain-solidity-course-py#1102] [Q&A] [answered]
Hi guys, when I run **brownie accounts new good-account** and **brownie test** result in nothing, but **brownie accounts generate good-account** actually works. I'm so confused now. Could anyone help me with this? I'm using Windows10. Thanks very much in advance!!!

![image](https://user-images.githubusercontent.com/46648495/154804034-de456ccc-a548-47ed-a025-43513f97a194.png)

**Comment by cromewar** [comment, marked as answer]
Hello @Guanxinyuan 
`new` is for importing an account using a private key.
`generate` is to create a new a account for ganache itself.

Please refer to these links in order to find more info about brownie accounts:

https://eth-brownie.readthedocs.io/en/stable/account-management.html
https://eth-brownie.readthedocs.io/en/stable/core-accounts.html

**Reply by Guanxinyuan** [reply]
Thank you for your answer@[cromewar](https://github.com/cromewar)!

I have just checked the link you provided (and the tutorial video as well). Both resources say that when running **brownie accounts new <id>**, an inputting block should be popped out I will be asked to input the private key to make a new account. However, the case is that there is nothing popping out for me to input anything. Similar problem applies to **brownie test**. It's also very weird that other brownie commands work pretty well as expected.

**Reply by cromewar** [reply]
Remember you should provide an ID: Means the name of the account you want to use, for example
![image](https://user-images.githubusercontent.com/20199180/154978196-1bcb67d0-dc09-4517-a58b-11c99a077225.png)

**Reply by Pato-wp** [reply]
Still not working same problem as before

**Reply by cromewar** [reply]
@Pato-wp please share more info on the problem, what you have tried so far and add screenshots.
